user-profiles.md

```diff
@@ -0,0 +1,36 @@
+# User Profiles
+
+The purpose of this document is to aide in the development of Helm features by evaluating Helm features against those whole will be using them.
+
+When considering requirements and implementation solutions it's useful look at who uses Helm to have a better understanding of how something should work. The Helm user profiles describe the different types of users and contributors to Helm along with the order of priority they have relative to each other. The ordering is because nothing can share the exact same priority and we want to convey that.
+
+## Preface
+
+* What's described here are user profiles as opposed to [personas](https://en.wikipedia.org/wiki/Persona#In_user_experience_design). Personas are example actors rather than general categories.
+* Kubernetes Cluster Operators are out of scope for this document. A cluster operator is one who manages the operation of a Kubernetes cluster that applications can run in.
+
+## Profiles
+
+Profiles describe a type of user. A real person may perform more than one type of role.
```

Comment:
It might not be completely evident here that "role" and "profile" are necessarily referring to the same profiles you mention below. To reduce any chance for confusion, it might be better to stick with the same terminology (i.e. profile instead of role). Maybe something like -- "In reality, multiple profiles may apply to the same person."

Reply:
Thanks for the feedback. I tried to clarify this. Let me know if it looks ok.